.github/workflows/ci-tests.yaml

```diff
@@ -49,6 +49,46 @@ jobs:
       - name: Upload to Codecov
         uses: codecov/codecov-action@v4
 
+  test_slow:
+    name: "Test Slow"
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ["ubuntu-latest"]
+        python-version: ["3.12"]
```

Comment:
I suggest we just use the latest version for the slow tests. WDYT?

Reply:
Yes, sounds good to me!